### DISCUSSION TITLE
what's the best solution on mobile browser which has "dynamic address bar"?

### DISCUSSION THREAD

**Discussion by youweol-master** [thebuilder/react-intersection-observer#597] [General]
I'm now  implementing a InfiniteList that use this library to trigger a fetchNextPage event.

I placed a zero-height HTML DIV block in the bottom of the list as the ref of inView trigger.

It works well on desktop browser and mobile browser that had a fixed address bar. Somehow, this did't work on the mobile browser has "dynamic address bar". what I mean about the browser has "dynamic address bar" is: while scroll-top the screen, the address bar hidden,
while scroll-down the screen, the address bar show. for example Firefox Moblie is one of them.  

when I set the HTML DIV a bigger size, about 100px (this will bigger than the address bar), the inView will work well too, but this solution is too ugry.

Is there a nice solution exists?

**Comment by thebuilder** [comment]
What doesn't work about it? Doesn't it trigger? Or does it trigger too often?

**Comment by arshamg** [comment]
Did you find a good solution here? curious if there's a solution that works across all mobile devices
